request-example.go

```diff
@@ -82,9 +85,7 @@ func (fs *root) putfile(pathname string, file *memFile) error {
 	return nil
 }
 
-func (fs *root) openfile(pathname string, flags uint32) (*memFile, error) {
-	pflags := newFileOpenFlags(flags)
-
+func (fs *root) openfile(pathname string, pflags FileOpenFlags) (*memFile, error) {
```

Comment:
This function signature had to change because the `NewFileOpenFlags` wasn’t being exposed anymore. But it makes sense that this function should take the output of the `Request.Pflags()` rather than delaying that transform until we get here.